data/tlg0064/tlg001/tlg0064.tlg001.1st1K-grc1.xml

```diff
@@ -0,0 +1,197 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="lat">Periplus Hannonis</title>
+                <author>Hanno</author>
+                <editor xml:lang="lat">CAROLUS MULLERUS</editor>
```

Comment:
Make sure that there are all caps words in the text elements of the teiHeader. These are usually used to draw attention to things that need to be changed.